swift/ios-shoppe-demo/APIService.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  APIService.swift
+//  ios-shoppe-demo
+//
+//  Created on 3/4/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
+import Foundation
+
+class APIService {
+    static var sharedInstance = APIService()
+
+    var baseURL: String {
+        guard let webserviceURL = getPlist(withName: "Preferences") else {
+            print("No web services URL found")
+            return ""
+        }
+        return webserviceURL[0]
+    }
+    
+    var products: [[String : Any]] = [] // TODO: Change this to be an array of class Product
+
+    func getShoppeItem(completion: @escaping()->()) {
+        guard let url = URL(string: baseURL) else { return }
+
+        let request = URLRequest(url: url)
+
+        URLSession.shared.dataTask(with: request) { (data, response, error) in
+            guard let data = data else { return }
+
+            do {
+                let json = try JSONSerialization.jsonObject(with: data, options: .mutableContainers) as? [Any]
+
+                for dict in json ?? [""] {
+                    self.products.append(dict as? [String : Any] ?? [:]) // TODO: use class Product
+                }
+                completion()
+            }
+            catch {
+                print("\(error.localizedDescription)") // TODO: Add proper error handling
```

Comment:
Please address these TODOs in your next PR.

Reply:
Sounds good I will do that